soc/arm/st_stm32/stm32l1/Kconfig.soc

```diff
@@ -9,6 +9,8 @@ choice
 prompt "STM32L1x MCU Selection"
 depends on SOC_SERIES_STM32L1X
 
+config SOC_STM32L151X8A
+	bool "STM32L151X8A"
 config SOC_STM32L151XB
```

Comment:
Break a line here. Sorry, I missed it earlier.

Reply:
sure, felt nice to make this boilerplate compact, but no problem.